### ISSUE TITLE
Build using cmake file

### ISSUE THREAD

**Issue by SaeedZhiany** [chenxin415/flowstar#1]
Hi,

Does any cmake file exist that I can use to build this project in IDEs like Clion?

**Comment by chenxin415** [comment]
Hi,

Thank you for using the tool. Sorry, I am not familiar with cmake, but I
guess that you may include the source code easily if all the dependencies
are installed.

best,
Xin

**Comment by SaeedZhiany** [comment]
Thank you for your response.

Cmake is a new branded procedure for compiling c++ projects and IDEs like Clion (by jetbrains) only support Cmake based projects. 

your current project is compilable only with eclipse or by `make` command from a terminal. I tried to convert your `MakeFile` to a `CMakeList.txt` but I still get some errors that I don't know how to resolve them. currently I have no access to my laptop. I try to share my `CMakeLists.txt` file with you to adding cmake support to your project.

**Comment by SaeedZhiany** [comment]
have you changed any of the original [FlowStar tool](https://flowstar.org/)?
what is the difference between this tool and the original one? what are the new features?
is this project for your research project? and if yes can you share your paper?

**Comment by chenxin415** [comment]
Yes, the version on github is entriely different from the one on the
homepage, since this version is only for providing C++ API on continuous
dynamics.

I use this version in all of the recent papers on the reachability analysis
for neural network controlled systems.

**Comment by SaeedZhiany** [comment]
So interesting, Can you please give me the link of your papers? I like to take a look at those.

and is your tool suitable for reachability analysis of Cyber-Physical Systems? or it's better to use the original one for CPSs.

**Comment by chenxin415** [comment]
Sure, you may check the following papers:

http://www.cs.colorado.edu/~srirams/papers/neural-reachability-hscc2019.pdf

https://arxiv.org/abs/1906.10654

Yes, Flow* is developed to analyze the safety of CPS, but our models are
just hybrid automata.

**Comment by SaeedZhiany** [comment]
Thank you.

> but our models are just hybrid automata.

sorry, I didn't get it.
